--- gtk-gnutella-0.96.1/extra_files/gtk-gnutella.desktop~	2006-02-22 17:50:20.000000000 +0100
+++ gtk-gnutella-0.96.1/extra_files/gtk-gnutella.desktop	2006-10-09 12:20:16.968893500 +0200
@@ -8,6 +8,7 @@
 Name[ja]=Gtk-Gnutella
 Name[nb]=Gtk-Gnutella
 Name[nl]=Gtk-Gnutella
+Name[pl]=Gtk-Gnutella
 Comment=A GUI based Gnutella Servent
 Comment[de]=Ein grafischer Gnutella-Servent
 Comment[fr]=Un servent Gnutella graphique
@@ -15,6 +16,7 @@
 Comment[ja]=GUI ベースの Gnutella サーバントです
 Comment[nb]=En grafisk Gnutella-tjenent
 Comment[nl]=Een grafische Gnutella servent
+Comment[pl]=Klient sieci Gnutella
 GenericName=P2P Client
 GenericName[de]=P2P-Client
 GenericName[fr]=Client P2P
@@ -22,6 +24,7 @@
 GenericName[ja]=P2P クライアント
 GenericName[nb]=Fildelingsprogram
 GenericName[nl]=P2P programma
+GenericName[pl]=Klient P2P
 TryExec=gtk-gnutella
 Exec=gtk-gnutella
 StartupNotify=false
