--- gtk-gnutella-0.96.3/extra_files/gtk-gnutella.desktop	2006-11-10 00:01:22.000000000 +0100
+++ gtk-gnutella-0.96.3/extra_files/gtk-gnutella.desktop	2007-03-01 13:40:53.000000000 +0100
@@ -9,6 +9,7 @@
 Name[nb]=Gtk-Gnutella
 Name[nl]=Gtk-Gnutella
 Name[nn]=Gtk-Gnutella
+Name[pl]=Gtk-Gnutella
 Comment=A GUI based Gnutella Servent
 Comment[de]=Ein grafischer Gnutella-Servent
 Comment[fr]=Un servent Gnutella graphique
@@ -17,6 +18,7 @@
 Comment[nb]=En grafisk Gnutella-tjenent
 Comment[nl]=Een grafische Gnutella servent
 Comment[nn]=Ein grafisk Gnutella-tenent
+Comment[pl]=Klient sieci Gnutella
 GenericName=P2P Client
 GenericName[de]=P2P-Client
 GenericName[fr]=Client P2P
@@ -25,6 +27,7 @@
 GenericName[nb]=Fildelingsprogram
 GenericName[nl]=P2P programma
 GenericName[nn]=Fildelingsprogram
+GenericName[pl]=Klient P2P
 TryExec=gtk-gnutella
 Exec=gtk-gnutella
 StartupNotify=false
