--- gtk-gnutella-GTK2-0.91.9/configure.in	Sat Feb  1 22:39:07 2003
+++ gtk-gnutella-GTK2-0.91.9-/configure.in	Tue Feb 25 15:23:47 2003
@@ -31,15 +31,6 @@
   enable_gtk2=no)
 
 
-if  test "x$enable_gtk2" = "xno"
-then
-    AM_PATH_GTK(1.2.0, ,
-	AC_MSG_ERROR(Cannot find GTK: Is gtk-config in path?))
-
-    AM_PATH_GLIB(1.2.10, ,
-			AC_MSG_ERROR(Cannot find GLib: Is glib-config in path?))
-    frontend="Gtk 1.2"
-else
     PKG_CHECK_MODULES(GTK_GNUTELLA, gtk+-2.0 >= 2.0.0 glib-2.0 >= 2.0.0)
     AC_SUBST(GTK_GNUTELLA_CFLAGS)
     AC_DEFINE(USE_GTK2)
@@ -49,7 +40,6 @@
 dnl We don't need any deprecated glib functions 
 	CPPFLAGS="${CPPFLAGS} -DG_DISABLE_DEPRECATED"
     
-fi
 AM_CONDITIONAL(ENABLE_GTK2, test "x$enable_gtk2" = "xyes")
 
 
