diff -Nur gtk-gnutella-0.92c.orig/configure.in gtk-gnutella-0.92c.chng/configure.in
--- gtk-gnutella-0.92c.orig/configure.in	Wed Apr 16 11:49:51 2003
+++ gtk-gnutella-0.92c.chng/configure.in	Mon Jul  7 12:42:19 2003
@@ -27,34 +27,19 @@
 AC_ARG_ENABLE(remote-shell,[
   --enable-remote-shell           provide a shell on the gnutella port (default=off)], , 
   enable_remote_shell=no)
-AC_ARG_ENABLE(gtk2,[
-  --enable-gtk2                   build a gtk2 version (default=off)], , 
-  enable_gtk2=no)
 AC_ARG_ENABLE(tos,[
   --enable-tos                    use IP Type of Service (default=on)], ,
 	enable_tos=yes)
 
 
-if  test "x$enable_gtk2" = "xno"
-then
-    AM_PATH_GTK(1.2.0, ,
-	AC_MSG_ERROR(Cannot find GTK: Is gtk-config in path?))
-
-    AM_PATH_GLIB(1.2.10, ,
-			AC_MSG_ERROR(Cannot find GLib: Is glib-config in path?))
-    frontend="Gtk 1.2"
-else
-    PKG_CHECK_MODULES(GTK_GNUTELLA, gtk+-2.0 >= 2.0.0 glib-2.0 >= 2.0.0)
-    AC_SUBST(GTK_GNUTELLA_CFLAGS)
-    AC_DEFINE(USE_GTK2, 1, [Define if you want to use GTK+ 2.x.])
+PKG_CHECK_MODULES(GTK_GNUTELLA, gtk+-2.0 >= 2.0.0 glib-2.0 >= 2.0.0)
+AC_SUBST(GTK_GNUTELLA_CFLAGS)
+AC_DEFINE(USE_GTK2, 1, [Define if you want to use GTK+ 2.x.])
 
-    frontend="Gtk 2.0"
+frontend="Gtk 2.0"
 
 dnl We don't need any deprecated glib functions 
-	CPPFLAGS="${CPPFLAGS} -DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED"
-    
-fi
-AM_CONDITIONAL(ENABLE_GTK2, test "x$enable_gtk2" = "xyes")
+CPPFLAGS="${CPPFLAGS} -DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED"
 
 
 dnl Check whether libresolv is needed for inet_aton()
diff -Nur gtk-gnutella-0.92c.orig/src/Makefile.am gtk-gnutella-0.92c.chng/src/Makefile.am
--- gtk-gnutella-0.92c.orig/src/Makefile.am	Sun May 18 11:37:35 2003
+++ gtk-gnutella-0.92c.chng/src/Makefile.am	Mon Jul  7 12:52:51 2003
@@ -25,8 +25,6 @@
 
 ## Process this file with automake to produce Makefile.in
 
-
-if ENABLE_GTK2
 INCLUDES = \
 	@GTK_GNUTELLA_CFLAGS@ \
 	@XML_CFLAGS@
@@ -53,40 +51,9 @@
 	upload_stats_gui2.c upload_stats_gui.h \
 	gnet_stats_gui2.c gnet_stats_gui.h \
 	search_stats_gui2.c search_stats_gui.h \
-	fileinfo_gui2.c fileinfo_gui.h \
+	fileinfo_gui.c fileinfo_gui.h \
 	main_gui.c main_gui.h main_cb.c main_cb.h 
 gtk_gnutella_LDADD = @INTLLIBS@ @GTK_GNUTELLA_LIBS@ -lz @XML_LIBS@
-else
-INCLUDES = \
-	@GTK_CFLAGS@ \
-	@XML_CFLAGS@
-GLADECMD = glade
-GLADEFILE = gtk-gnutella.glade
-GLADE_UI = \
-	interface-glade1.c interface-glade1.h \
-	support-glade1.c support-glade1.h
-ADDITIONAL_DROPIN = \
-	filter.c filter.h filter_cb.c filter_cb.h filter_gui.c filter_gui.h \
-	glib-missing.c glib-missing.h \
-	gtk-missing.c gtk-missing.h \
-	gui.c gui.h \
-	callbacks.c callbacks.h \
-	gtkcolumnchooser.c gtkcolumnchooser.h \
-	monitor_gui.c monitor_gui.h monitor_cb.c monitor_cb.h \
-	nodes_gui.c nodes_gui.h nodes_cb.c nodes_cb.h \
-	gui_property.c gui_property.h gui_property_priv.h \
-	settings_gui.c settings_gui.h settings_cb.c settings_cb.h \
-	statusbar_gui.c statusbar_gui.h \
-	search_gui.c search_gui.h search_cb.c search_cb.h \
-	downloads_gui.c downloads_gui.h downloads_cb.c downloads_cb.h \
-	uploads_gui.c uploads_gui.h uploads_cb.c uploads_cb.h \
-	upload_stats_gui.c upload_stats_gui.h \
-	gnet_stats_gui.c gnet_stats_gui.h \
-	search_stats_gui.c search_stats_gui.h \
-	main_gui.c main_gui.h main_cb.c main_cb.h \
-	fileinfo_gui.c fileinfo_gui.h
-gtk_gnutella_LDADD = @INTLLIBS@ @GTK_LIBS@ -lz @XML_LIBS@
-endif
 
 bin_PROGRAMS = gtk-gnutella
 CLEANFILES = *.bak
